docs/experiments-clueweb09b.md

```diff
@@ -5,11 +5,14 @@ Indexing:
 ```
 nohup sh target/appassembler/bin/IndexCollection -collection CW09Collection \
  -input /path/to/cw09/ClueWeb09_English_1/ -generator JsoupGenerator \
- -index lucene-index.cw09b.pos+docvectors -threads 32 -positions -docvectors -optimize \
+ -index lucene-index.cw09b.pos+docvectors -threads 32 -storePositions -storeDocvectors -optimize \
```

Comment:
@Peilin-Yang yes, we have been using these commands ever since the code was updated but it's just that we hadn't updated the docs. Also, I have done a separate regression testing on a bunch of the collections. If it looks good to you, please approve it...